src/palindrom.js

```diff
@@ -330,7 +376,9 @@ var Palindrom = (function() {
       onSocketOpenCallback && onSocketOpenCallback(event);
     };
     that._ws.onmessage = function(event) {
-      that.onReceive(JSON.parse(event.data), that._ws.url, 'WS');
+      const parsedMessage = JSON.parse(event.data);
+      that.validateNumericsRangesInPatch(parsedMessage, 'incoming');
+      that.onReceive(parsedMessage, that._ws.url, 'WS');
```

Comment:
Why not putting `validateNumericsRangesInPatch` in `onReceive` for DRYness?

Was a call to `validateNumericsRangesInPatch` not added in `PalindromNetworkChannel.prototype.send` on purpose?